.github/workflows/build_binaries_on_new_releases.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           os: macos-latest
           target: x86_64-apple-darwin
         - name: windows
-          os: windows-2019
+          os: windows-2022
```

Comment:
Why do we use windows-2022 rather than windows-latest? As I can see, `.github/workflows/bot.yml` use windows-latest also other platform use `-latest` as well.

Reply:
Thank you for mentioning it. Because I believe that a stable binary generation is preferable, I specify `windows-2022` instead of `windows-latest`. However, I did not notice that other platforms use the latest version. It seems that changing to `latest` is preferable for unification. Thus, I modified it as `window-latest`. I have created an issue to discuss the preferable version to support at #186.
On the other hand, I do not have a good reason for using `windows-latest` in  `bot.yml`. Do you have any opinion regarding `bot.yml`?

Reply:
From my opinion, we need to unify image on all of workflow. Because, the snapshot is actually not to relate building binary itself but if the image is different we are hard to identify the cause when any issue occur.
So would you mind if we use `windows-2022` instead of `windows-latest` on other workflow for now and will discuss it on the issue?

Reply:
Thanks. I think your opinion makes sense. Due to the scope of this PR, I changed `os` to `windows-2022` for both CI and bots for the Windows platform. Let's discuss this within #186 about other platforms.